### PR TITLE
add API endpoint for include files reload

### PR DESCRIPTION
**Pull request by vladak**
Tested with:

- create `footer_include` file
- deploy webapp
- check the footer is displayed
- change include file content
- reload:
```shell
curl -v -X PUT http://localhost:8080/source/api/v1/system/includes/reload
```
- check the changed footer is displayed

### CODE REVIEW

**Comment by tulinkry** [comment]
update the wiki

**Comment by vladak** [comment]
Sure, https://github.com/oracle/opengrok/wiki/Web-services will be updated as soon as this merges in.

**Comment by vladak** [comment]
For the test I had to use the `RestApp` magic otherwise Grizzly fails when resolving the suggester dependency for `SystemController`.

File: opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SystemControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        super.tearDown();
+    }
```
Comment (tulinkry) [inline opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SystemControllerTest.java]: are these two needed?

Comment (vladak) [reply opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SystemControllerTest.java]: Looks like not.

**Pull request merged**